duvet/src/extract.rs

```diff
@@ -139,17 +193,17 @@ fn extract_section(section: &Section) -> (&Section, Vec<Feature>) {
 
         if KEY_WORDS_SET.is_match(line) {
             for (key_word, level) in KEY_WORDS.iter() {
-                for occurance in key_word.find_iter(line) {
+                for occurrence in key_word.find_iter(line) {
```

Comment:
guess we need to add the typo check to CI

Reply:
Haha yep. I added it to my entire and found a few issues 😁